src/test/FuzzerImpl.cpp

```diff
@@ -984,7 +985,8 @@ applyFuzzOperations(LedgerTxn& ltx, PublicKey const& sourceAccount,
 {
     auto txFramePtr = createFuzzTransactionFrame(ltx, sourceAccount, begin, end,
                                                  app.getNetworkID());
-    txFramePtr->attemptApplication(app, ltx);
+    TransactionResult txRes;
```

Comment:
Since the result is not used anywhere, would it make sense to just allocate `txRes` in `attemptApplication`?